modules/custom/az_card/src/Plugin/Field/FieldFormatter/AZCardDefaultFormatter.php

```diff
@@ -239,6 +237,7 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
           '#langcode' => $item->getLangcode(),
         ],
         '#link' => $link_render_array,
+        '#link_url' => $this->pathValidator->getUrlIfValid($item->link_uri ?? '<none>'),
```

Comment:
link_url, should this be link_uri for the new twig variable? I haven't tried this yet, just looking.

Reply:
Didn't work for me :(

Reply:
Ok, I went to test and confirmed: this should be `#link_uri` instead of `#link_url`.

Also, it might be best to use what $link_render_array is using on line 137 just to make sure it's the same as the original rendered link.
That would change this line to:
`'#link_uri' => $link_url ?: Url::fromRoute('<none>'),`